src/Http/samples/MinimalSample/Program.cs

```diff
@@ -1,10 +1,15 @@
 using System;
 using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Hosting;
 
-await using var app = WebApplication.Create();
+await using var app = WebApplication.Create(args);
```

Comment:
I'm just now appreciating that this removes any ability to configure DI, options, etc., those require the builder. What's the simplest version of this that makes the builder discoverable?
```C#
var builder = WebApplication.CreateBuilder(args);
await using var app = builder.Build();
```
That extra line seems worth it.

Reply:
This is just the sample. I'm guessing you want to change this in the template as well. @LadyNaggaga thoughts?

Reply:
Can people actually do anything interesting without configuring DI/options?

Reply:
With WebApplicaiton they can do middleware, routes, and IConfiguration, but not set up logging, options, DI, servers, etc.

Reply:
Then we just need a magic autobind so you can just set options by convention in config!...kidding

Reply:
I dunno, maybe. We basically need to go against our natural instincts here 😄 

cc @LadyNaggaga

Reply:
We're adding the builder.